autoria.py

```diff
@@ -399,11 +399,26 @@ def select_item(item_to_select: str, items_list: dict) -> int:
             needed item (category, bodystyle, mark etc.) identifyer.
         """
         if item_to_select is not None and items_list is not None:
+            result = None
             for item in items_list:
-                if fnmatch(item['name'], '*' + item_to_select + '*'):
-                    return item['value']
-        else:
-            return None
+                if fnmatch(item['name'], item_to_select):
+                    result = item['value']
+                    print(result)
+                    return result
+            if result is None:
+                for item in items_list:
```

Comment:
Do we need multiple for - in loops? Can we have one loop and multiple checks inside it?
Or you actually need it check through all items before moving to another search method?